include/functions.h

```diff
@@ -171,6 +173,7 @@ s32 phys_can_player_interact(void);
 void ai_enemy_play_sound(Npc* npc, s32 arg1, s32 arg2);
 
 s32 player_test_move_without_slipping(PlayerStatus*, f32*, f32*, f32*, s32, f32, s32*);
+s32 player_test_move_with_slipping(PlayerStatus*, f32* posX, f32* posY, f32* posZ, f32 Speed, f32 Heading);
```

Comment:
This also produces a warning w.r.t. the first argument being unnamed. Also `Heading` should probably be `heading` (no capital)?

Reply:
I assume the same goes for speed